boards/arm64/xenvm/xenvm_gicv3_defconfig

```diff
@@ -1,20 +1,44 @@
 CONFIG_SOC_XENVM=y
 CONFIG_BOARD_XENVM=y
+CONFIG_XEN_INITIAL_DOMAIN=y
 
 # Use GICv3
 CONFIG_XENVM_USE_GIC_V3=y
 
 # Enable UART driver
 CONFIG_SERIAL=y
 
-CONFIG_MAX_XLAT_TABLES=10
+CONFIG_MAX_XLAT_TABLES=24
+CONFIG_HEAP_MEM_POOL_SIZE=2097152
 
 # Enable console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
+# Enable PV hypervisor console
+CONFIG_UART_XEN_HVC_CONSOLEIO=y
+
 # Enable logging subsys
 CONFIG_LOG=y
 CONFIG_LOG_MODE_MINIMAL=n
 
-CONFIG_USERSPACE=n
+CONFIG_TEST=y
```

Comment:
(question) could you please move all below config to application config file?
do we really need them here?

Reply:
Sure I can. But I've just copied it from xenvm_defconfig. If moving something to app - then all configs should be updated. Why did those parameters where added to xenvm and xenvm-qemu in the first place?

Reply:
I think it was a mistake and probably because people want to have fast crutch solutions

Reply:
somebody don't want to use this config or for example shell config, why should board have it for all samples/applications?
I believe that board's configs may contain only minimal configuration 
board's configs should be universal and other stuff should be handled by configs of apps/samples/tests